controllers/servingruntime_validator.go

```diff
@@ -17,6 +17,8 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
NIT: the spacing in this import block is strange; I think `"errors"` should be up with the other stdlib packages and this down with the other `k8s.io` import.

Reply:
@tjohnson31415 updated, PTAL